pkg/kv/kvserver/rangefeed/event_size_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 The Cockroach Authors.
+// Copyright 2104 The Cockroach Authors.
```

Comment:
I like your optimism :D

Reply:
No idea how that got there

Reply:
Oh, I find-and-replaced 202 with 210 to add 8 bytes.